src/Layer/TiledGeometryLayer.js

```diff
@@ -192,7 +192,7 @@ class TiledGeometryLayer extends GeometryLayer {
         }
 
         // do proper culling
-        node.visible = (!this.culling(node, context.camera));
+        node.visible = this.culling(node, context.camera);
```

Comment:
why you inverse?

Reply:
There was an error, but it was because of the `this.culling = () => true` in `GeometryLayer`, that I removed. It shouldn't have been a placeholder.